### ISSUE TITLE
Configurable padding

### ISSUE THREAD

**Issue by Mark-Joy** [termux/termux-app#2210]
**Feature description**

I know Termux has [a little padding](https://github.com/termux/termux-app/issues/516), can we make this padding configurable?
My phone has its screen edged covered a little bit. I think some people also have the same issue.

![IMG_20210806_170913](https://user-images.githubusercontent.com/22109528/128495456-5b517b72-c88e-45c0-9374-f5fd93c12125.jpg)

**Comment by Aridlvory** [comment]
Yes it's a big problem for me... I shuld tilt the phone or open recent apps to check what's at the edge.    And it can be resolved by changing the padding level of the whole phone's os with the help of **developer options**
But I still follow with @Mark-Joy 's request because if we did anything wrong in the Dev options and we can't get back, Then we are stuck. But if we do something wrong in termux setting, we can get back by cp all files or not down the source of all resources... Delete termux.... Reinstall termux.... Then re-download or cp all the files
Changing the os setting is bad...

**Comment by Mark-Joy** [comment]
@Aridlvory 
The solution was already implemented, just wait for the next release.
